### PR TITLE
Improvements to build & troubleshooter script

### PR DESCRIPTION
**Pull request by fredyvilla**

### CODE REVIEW

File: build_minecraft_bot.sh [hunk 1]
```diff
@@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
 else
     cd catkin_ws/src
     # Add the package
-    ln -s ../../minecraft_bot/ minecraft_bot
+    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
+    cp -R ../../minecraft_bot .
+    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
```
Comment (AmeBel) [inline build_minecraft_bot.sh]: what do yo mean by symlinks don't work in virtualbox?

Comment (fredyvilla) [inline build_minecraft_bot.sh]: When using vagrant, the symlink command gives this error:

ln: failed to create symbolic link ‘minecraft_bot’: Protocol error

This happens when using a shared folder in virtualbox.

Found a fix I'm testing:
http://www.ahtik.com/blog/fixing-your-virtualbox-shared-folder-symlink-error/


Give me 5 mins..

File: build_minecraft_bot.sh [hunk 2]
```diff
@@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
...
 fi
 
+echo "building through catkin_make"
 cd catkin_ws
 catkin_make
+
+echo "updating opencog_python_eval.conf to catkin_ws path"
+cd src/minecraft_bot/src
+cwd=$(pwd)
+sed -ie "s#CATKIN_WS_PATH#$cwd#" opencog_python_eval.conf
```
Comment (AmeBel) [inline build_minecraft_bot.sh]: please update the readme for the steps.

File: build_minecraft_bot.sh (hunk 1 above)
Comment (AmeBel) [inline build_minecraft_bot.sh]: If the Vagrantfile is particular to this project would you pls include it in the pr?

Comment (fredyvilla) [inline build_minecraft_bot.sh]: OK I figured out a fix.  Using the vagrant file in opencog/opencog

You need to run virtualbox as administrator
 and then run this at cmd:
VBoxManage setextradata opencog-dev-vm
VBoxInternal2/SharedFoldersEnableSymlinksCreate//vagrant 1


Send a new commit to put back the symlink.

**Comment by fredyvilla** [comment]
Will update readme tomorrow

File: build_minecraft_bot.sh (hunk 1 above)
Comment (fredyvilla) [inline build_minecraft_bot.sh]: done with changes, please merge PR

**Comment by AmeBel** [comment]
Thanks

**Comment by AmeBel** [comment]
The less steps to get started the better, this removes one more step :smile:

**Pull request merged**